Packs/AWS-EC2/Scripts/AwsEC2GetAccounts/AwsEC2GetAccounts.py

```diff
@@ -0,0 +1,72 @@
+import demistomock as demisto
+from CommonServerPython import *
+
+
+ACCOUNT_LIST_COMMAND = 'aws-org-account-list'
+EC2_ACCOUNTS_PARAM = 'accounts_to_access'
+
+
+def internal_request(method: str, uri: str, body: dict = {}) -> dict:
+    return json.loads(demisto.internalHttpRequest(method, uri, body)['body'])
+
+
+def get_account_ids() -> list[str]:
+    try:
+        account_list_result: list[dict[str, dict]] = demisto.executeCommand(ACCOUNT_LIST_COMMAND, {})  # type: ignore
+        return [
+            account['Id']
+            for account
+            in account_list_result[0]['EntryContext']['AWS.Organizations.Account(val.Id && val.Id == obj.Id)']
+        ]
+    except ValueError:
+        raise DemistoException(f'The command {ACCOUNT_LIST_COMMAND!r} must be operational to run this script.')
```

Comment:
Do we have the server output? We should print this. (you dont have an instance enabled or whatver message they print)